src/sentry/replays/testutils.py

```diff
@@ -138,7 +138,7 @@ def mock_replay(
                         "segment_id": kwargs.pop("segment_id", 0),
                         "tags": tags,
                         "urls": kwargs.pop("urls", []),
-                        "is_archived": kwargs.pop("is_archived", False),
+                        "is_archived": kwargs.pop("is_archived", None),
```

Comment:
this line doesn't actually matter as in our snuba consumer we will only ever set is_archived to `1` or `None` can change back if preferred.